README.md

```diff
@@ -1,107 +1,64 @@
+It is built from the boilerplate [React Cool Starter](https://github.com/wellyshen/react-cool-starter), on top of [React](https://facebook.github.io/react), [Redux](https://github.com/reactjs/redux), [React Router](https://reacttraining.com/react-router), [Express](https://expressjs.com), [Yarn](https://yarnpkg.com/), [webpack](https://webpack.js.org/) and [Babel](https://babeljs.io/).
 
-## Requirements
+![Illustration of the website](.github/screenshot.png)
 
+## Contributing
+
+### Onboarding
+
+This project is open to contribution, but you will likely need to be in touch with us in order to collaborate efficiently. Contact Paris Est Ludique via our official Website then join the Discord. You will need to get:
+
+-   Access to a database sample to place in `./access/dbToLoad.json`
+-   Access to our [Trello Issue trackerd](https://trello.com/b/5oKGdc1K/intranet-pel-2022)
+
+### Requirements
+
+-   An account on GitHub
+-   [git](https://git-scm.com/)
 -   [node](https://nodejs.org/en) >= 12.0
 -   [npm](https://www.npmjs.com) >= 6.0
+-   On MacOS, [HomeBrew](https://brew.sh/) is recommanded
+-   An IDE of your choice. We suggests:
+    -   [Atom](https://atom.io/) with [TypeScript plugin](https://atom.io/packages/ide-typescript)
+    -   [Visual Studio Code](https://code.visualstudio.com/)
+    -   [Webstorm 2018.1](https://www.jetbrains.com/webstorm/download/)
+    -   [Sublime Text](http://www.sublimetext.com/3) with [Typescript-Sublime-Plugin](https://github.com/Microsoft/Typescript-Sublime-plugin#installation)
 
-## Getting Started
+### Setup
 
-**1. You can start by cloning the repository on your local machine by running:**
+Using a terminal:
 
-```sh
+```bash
+# Clone this repository on your computer
 git clone https://github.com/forceoranj/intranet.git
 cd intranet
-```
 
-**2. Install all of the dependencies:**
-
-```sh
-npm install yarn
+# Install dependencies using YARN
+npm install --legacy-peer-deps
+brew install corepack # If not included in your npm
+corepack enable
 yarn
-```
 
-**3. Start to run it:**
+# Include the database
+mv ~/Downloads/database-sample.json ./access/dbToLoad.json
 
-```sh
+# Start your local server
 yarn dev  # Build, hosts, and hot reload saved modifications
 ```
 
-Now the app should be running at [http://localhost:3000](http://localhost:3000)
+Now the app should be serving at <http://localhost:3000>.
 
-## File editors
+### Contribution rules
 
-Edit files with one of these HTML/CSS/TypeScript editors:
+We stick to the [GitHub Flow workflow](https://docs.github.com/en/get-started/quickstart/github-flow)
 
--   [Atom](https://atom.io/) with [TypeScript plugin](https://atom.io/packages/ide-typescript)
--   [Visual Studio Code](https://code.visualstudio.com/)
--   [Webstorm 2018.1](https://www.jetbrains.com/webstorm/download/)
--   [Sublime Text](http://www.sublimetext.com/3) with [Typescript-Sublime-Plugin](https://github.com/Microsoft/Typescript-Sublime-plugin#installation)
-
-## Script Commands
```

Comment:
It will be great to keep the **script commands** and the **app structure** sections for the moment ? before making a better documentation :)

Reply:
Let's add it back :)